tests/acceptance/Admin/AdminLoginHappyPath.feature

```diff
@@ -0,0 +1,10 @@
+Feature: AdminLoginHappyPath
+  As admin user
+  I want to make a successful login to Woocommerce administration interface
+
+  @woocommerce @test
+  Scenario: Admin login to administration interface
+    Given I initialize shop system
+    When I login to admin interface as "admin user"
+    Then I see "Howdy, admin" on screen
+    
```

Comment:
same as before, there should be an empty line in the end of the file

Reply:
I've added empty line (line 10) to AdminLoginHappyPath.feature file in previous commit.

Reply:
ok, sorry, github still highlighting that for me. weird. Do you see a sign saying "No newline at the end of the file"?

Reply:
Yes, I see it, along with newline beyond it... Really strange. :-)